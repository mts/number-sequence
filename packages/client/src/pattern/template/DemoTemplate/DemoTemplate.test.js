import React from 'react'
import demoTemplateDefault from './DemoTemplate.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

jest.mock('react-router-dom', () => {
  return {
    Link: () => <div />,
  }
})

describe('<DemoTemplate />', () => {
  describe('Snaphot', () => {
    test('must match demoTemplateDefault', () => {
      expect(global.renderToJSON(demoTemplateDefault)).toMatchSnapshot()
    })
  })
})
