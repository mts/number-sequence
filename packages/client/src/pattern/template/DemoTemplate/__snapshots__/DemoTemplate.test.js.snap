// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DemoTemplate /> Snaphot must match demoTemplateDefault 1`] = `
<div>
  <h1
    align="center"
    className="p-4 text-red"
  />
  <div>
    <div />
    <div
      align="center"
      className="p-4"
    >
      <div />
    </div>
  </div>
</div>
`;
